publish-release.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
```

Comment:
can we put this script in some subfolder? to cut down on random files in the root dir